index.js

```diff
@@ -6,23 +6,43 @@ var async = require('async');
 var mkdirp = require('mkdirp');
 var iferr = require('iferr');
 var spundle = require('spundle');
+var Options = require('options');
```

Comment:
Also, stylistically, this variable name should not start with an uppercase letter.